### PR TITLE
Updates to the Data Access Overview notebook

### PR DESCRIPTION
**Pull request by jacquesalice**

### CODE REVIEW

File: 02_DataAccessOverview/DataAccessOverview.ipynb [hunk 1]
```diff
@@ -317,24 +568,35 @@
...
    "source": [
-    "fig, ax = plt.subplots(figsize=(5,5))\n",
+    "fig, ax = plt.subplots(figsize=(10,10))\n",
     "x = df['g_r']\n",
     "y = df['gmag']\n",
-    "ax.scatter(x,y,marker='.',s=10, alpha=0.8)\n",
-    "ax.set_xlabel(x.name)\n",
-    "ax.set_ylabel(y.name)\n",
+    "ax.scatter(x,y,marker='.',s=10, alpha=0.8) # s = size, alpha = transparency of data points\n",
+    "ax.set_xlabel('g-r',fontsize=15)\n",
```
Comment (rnikutta) [inline 02_DataAccessOverview/DataAccessOverview.ipynb:575]: Propose 'g - r' (with extra spaces)

File: 02_DataAccessOverview/DataAccessOverview.ipynb [hunk 2]
```diff
@@ -317,24 +568,35 @@
...
+    "ax.set_ylabel('g magnitude',fontsize=15)\n",
```
Comment (rnikutta) [inline 02_DataAccessOverview/DataAccessOverview.ipynb:576]: Propose 'g (mag)'

File: 02_DataAccessOverview/DataAccessOverview.ipynb (hunk 1 above)
Comment (jacquesalice) [reply 02_DataAccessOverview/DataAccessOverview.ipynb]: Done.

File: 02_DataAccessOverview/DataAccessOverview.ipynb (hunk 2 above)
Comment (jacquesalice) [reply 02_DataAccessOverview/DataAccessOverview.ipynb]: Done.

**Pull request closed without merging**